types/email-templates/index.d.ts

```diff
@@ -84,7 +85,15 @@ declare class EmailTemplate {
      * @param view The Html pug to render
      * @param locals The template Variables
      */
-    render(view: string, locals: any): Promise<string>;
+    render(view: string, locals?: any): Promise<string>;
+    /**
+     * Render all available template files for a given email
+     * template (e.g. `html.pug`, `text.pug`, and `subject.pug`)
+     *
+     * @param view Name of the template
+     * @param locals The template variables
+     */
+    renderAll(view: string, locals?: any): Promise<EmailTemplate.EmailMessage>;
```

Comment:
remove EmailTemplate and just use EmailMessage, after the interface is moved with the others.